security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+For a precise definition of central point dominance, the reader is referred to https://doi.org/10.2307/3033543[Freeman's work].
+The larger the central point dominance of a network is, the more centralized the network is.
+We can observe that the LN has a greater central point dominance, ie. it is more centralized, than a random graph (Erdős-Rényi graph) or a scale-free graph (Barabási-Albert graph) of equal size.
+However, we also note, that over time the central point dominance of the LN gradually decreases as the network grows and densifies.
+
+[[central-point-dominance-ln]]
+.Central point dominance of the LN, a random graph (ER) and a scale-free graph (BA) of equal size.
+image:images/central-point-dominance.png[]
+
+In general, our understanding of the dynamic nature of the LN channel graph is rather limited.
+It is fruitful to analyze how protocol changes like multi-part payments can affect the dynamics of the LN.
+It would be beneficial to explore the temporal nature of the LN graph in more depth.
+
+
+=== Economic incentives and graph structure
+The Lightning graph forms spontaneously.
+Nodes connect to each other based on mutual interest.
+As a result, incentives drive graph development.
+Let's describe some of the relevant incentives:
+
+  * Rational incentives.
+    - Nodes establish channels to send, receive, and route payments (earn fees).
+    - What makes a channel more likely to be established between two nodes that act rationally?
+  * Altruistic incentives.
+    - Nodes establish channels "for the good of the network."
+    - While we should not base our security assumptions on altruism, to a certain extent, altruistic behavior drives Bitcoin (accepting incoming connections, serving blocks).
+    - What role does it play in Lightning?
+
+A recent paper has shown that LN nodes are mostly altruistic.
+Specifically, the earned routing fees do not compensate for the opportunity costs stemming from liquidity lock-up.
+This might change in the future if LN has significantly larger traffic or if a market for routing fees emerge.
+On the other hand, if a node wishes to optimize its routing fees, it would minimize the average shortest path lengths to every other node.
+Put differently, a profit-seeker node will try to locate itself in the _center_ of the channel graph or close to it.
+Given these incentives and the fact that capital is also unevenly distributed, it is unlikely that LN will evolve into a mesh-like network.
+
+= Practical advice for users to protect their privacy =
+
+We're still in the early stages of the Lightning Network.
+Many of the concerns listed in this chapter are likely to be addressed as it matures and grows.
+In the meantime, there are some measures that you can take to guard your node against malicious users; something as simple as updating the default parameters that your node runs with can go a long way in hardening your node.
+
+== Private channels
```

Comment:
Another benefit is that one is able to more effectively guard/utilize its channel bandwidth: given that these private channels will only be used to receiving or sending directly to that node.

Reply:
Thanks! Added your insight!